src/components/scene/wakelock.js

```diff
@@ -0,0 +1,15 @@
+var register = require('../../core/component').registerComponent;
+var Wakelock = require('../../../vendor/wakelock/wakelock');
```

Comment:
It seemed to make sense with the system. Components modify behaviors of whatever they attach to. There doesn't seem to be a rule about components that they have to be configurable or optional.